core/server/web/shared/middlewares/image/handle-image-sizes.js

```diff
@@ -4,6 +4,7 @@ const storage = require('../../../../adapters/storage');
 const activeTheme = require('../../../../services/themes/active');
 
 const SIZE_PATH_REGEX = /^\/size\/([^/]+)\//;
+const UNRESIZABLE_FILETYPE_REGEX = /\.(?:gif|svgz?)$/;
```

Comment:
Maybe we could unify it with same logic here: https://github.com/TryGhost/Ghost/blob/fcd275f6c098684ad615b02d0714cf38e5cd8f44/core/server/web/shared/middlewares/image/normalize.js#L14
Tbh, not a fan of regexes when it comes to blacklists like here or any data that can be represented as an array, as it's not visible from first glance that both `.svg` and `.svgz` would be caught by this regexp :thinking:

Reply:
Blacklist as an array deffo makes more sense :ok_hand: 

Not 100% sure what you meant by unify, so far I've copied the array - do you think we should export the file extension blacklist? Maybe from the `lib/image` or `middleware/image` module?

Reply:
Yes :+1: , think file extension blacklist should be a part of `lib/image/manipulator` and both middlewares - `normalize` & `handle-image-sizes` would be using the same method like `canTransform` or something along those lines

Reply:
Yeah nice idea :relaxed: I've added that now

Reply:
Looks great :+1: Just one more small thing, this note - https://github.com/TryGhost/Ghost/pull/10315/files#diff-ff55d92bb888edcce1b6466a7c7bbc65R10 , would need to be moved to this new method as well. Other then this, it looks very shippable :ship:

Reply:
Nice catch :male_detective: - updated!